package.json

```diff
@@ -35,7 +35,7 @@
     "@fortawesome/free-regular-svg-icons": "^5.12.0",
     "@fortawesome/free-solid-svg-icons": "^5.12.0",
     "@fortawesome/react-fontawesome": "^0.1.8",
-    "@gnosis.pm/dex-js": "@gnosis.pm/dex-js@^0.5.0",
+    "@gnosis.pm/dex-js": "0.6.1",
```

Comment:
The fix is included on version [`0.7.1-rc.0`](https://github.com/gnosis/dex-js/pull/190/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R3), no?

Reply:
We've done a hotfix in dex-js too, since there's a breaking change in 0.7.0

The fix is applied both in 0.6.1 and 0.7.1 but the web needs the compatible one
0.7.1 can be used only in develop

Reply:
Leandro, you are reading the PRs :) 
Come back officially!

Reply:
Well, I was tagged as reviewer, I'm reviewing it :shrug: